src/fidesops/graph/config.py

```diff
@@ -204,6 +203,21 @@ class Config:
 
         arbitrary_types_allowed = True
 
+    def cast(self, value: Any) -> Optional[Any]:
+        """Cast the input value into the form represented by data_type.
+
+        - If the data_type is None, then it has not been specified, so just return the input value.
+        - Return either a cast value or None"""
+
+        if self.data_type:
+            # if no data type is specified just return the input value.
+            # Skip conversions for query tokens, which are only for display output
+            if not isinstance(value, QueryToken):
+                converter: DataTypeConverter = self.data_type.value
+                return converter.to_value(value)
```

Comment:
Could we have a dev-friendly log if the data type isn't supported?

Reply:
Yes, but I think that should get caught in the validations on the dataset configuration in the endpoints.